website/static/js/tests/projectSettings.test.js

```diff
@@ -137,4 +139,34 @@ describe('ProjectSettings', () => {
             assert.called(resetMessageSpy);
         });
     });
+
+    describe('InstitutionSettings ViewModel', () => {
+        var viewModel = InstitutionSettingsViewModel();
+        var modifyStub;
+        var modifyInstStub;
+        var bootboxStub;
+
+        before(() => {
+            modifyInstStub = sinon.stub(viewModel, '_modifyInst');
+            bootboxStub = sinon.stub(bootbox, 'dialog');
+        });
+        after(() => {
+            viewModel.modifyChildrenDialog.restore();
```

Comment:
modifyChildrenDialog isn't stubbed; is this supposed to be `bootboxStub.restore()`?